api/brands/campaigns-api.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+const axios = require('axios');
+const baseApi = "https://api.flatlay.io";
+const faker = require("faker");
+const campaignName = faker.random.words(4);
+
+class CampaignsBrandsApi {
+get $selectorForToken() {return $(".brand-layout");} 
+get $apiCampaignName() {return $(`li=${campaignName}`);} 
```

Comment:
api page object can not OR should not have selectors, as API doesn't use the selector. You probably meant UI pom